### PR TITLE
fix:customize quotatiom items child doc

### PR DESCRIPTION
**Pull request by avishna-murali**
## Feature description
1. customize quotation items child doc 
## Solution description
1. customize quotation items child doc 
2. set default value 1 in quantity (quotation items child doc )
## Output
image

## Areas affected and ensured
-New feature

## Is there any existing behavior change of other features due to this code change?
-No

## Was this feature tested on the browsers?
  - Mozilla Firefox

### CODE REVIEW

**Review by MhmdSinanKT — CHANGES_REQUESTED** [review]
Don't do it like this, the qty should be set to 1 default while mapping the data from lead

File: versa_system/versa_system/custom_script/quotation.py [hunk 1]
```diff
@@ -17,9 +17,9 @@ def set_missing_values(source, target):
 
     def filter_approved_items(source, target, source_parent):
         # Only map rows where the 'approve' checkbox is checked
-        if source.approve:
```
Comment (MhmdSinanKT) [inline versa_system/versa_system/custom_script/quotation.py:20]: why was this removed?

**Review by MhmdSinanKT — CHANGES_REQUESTED** [review]

File: versa_system/versa_system/custom_script/quotation.py (hunk 1 above)
Comment (avishna-murali) [reply versa_system/versa_system/custom_script/quotation.py:20]: The 'ItemDetails' object has no attribute 'approve' so I removed that condition, there is no need for that condition

**Review by MhmdSinanKT — APPROVED** [review]
